src/main/java/org/broadinstitute/hellbender/tools/picard/vcf/VcfToIntervalList.java

```diff
@@ -23,6 +23,8 @@
         programGroup = VariantProgramGroup.class
 )
 public final class VcfToIntervalList extends PicardCommandLineProgram {
+    public static final String INCLUDE_FILTERED_LONG_NAME = "includeFiltered";
```

Comment:
Splitting hairs: might be clearer to move declaration of INCLUDE_FILTERED names to just before they are first used.